docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -30,13 +30,29 @@ NOTE: PacketFence PKI is available on the PacketFence repository as the package
 Installing 
 ^^^^^^^^^^
 
-First of all, you will need to install the package, use the following command matching your OS.
+First of all you have the choice to install the packetfence-pki on a stand alone machine or on the PacketFence machine.
```

Comment:
*PacketFence PKI
Don't name the package name when you're referring to the product name